include/deal.II/matrix_free/fe_point_evaluation.h

```diff
@@ -264,7 +269,7 @@ namespace internal
                    const unsigned int          vector_lane,
                    const scalar_gradient_type &result)
       {
-        for (unsigned int d = 0; d < dim; ++d)
+        for (unsigned int d = 0; d < spacedim; ++d)
```

Comment:
This looks wrong `scalar_gradient_type`  is `dim` long and you loop until `spacedim`?